drivers/include/bmx055.h

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_bmx055 BMX055 9-axis sensor
+ * @ingroup     drivers_sensors
+ * @brief       Device driver for the Bosch BMX055 9-axis sensor
+ * @{
+ *
+ * @file
+ * @brief       Device driver interface for the Bosch BMX055 9-axis sensor
+ *
+ * @note        The current state of the driver only implements basic polling.
+ *
+ * @author      Semjon Kerner <semjon.kerner@fu-berlin.de>
+ */
+
+#ifndef BMX055_H
+#define BMX055_H
+
+#include <stdint.h>
+
+#include "periph/i2c.h"
+#include "periph/gpio.h"
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name   The sensors default I2C addresses
+ * @{
+ */
+#define BMX055_MAG_ADDR_DEFAULT  0x10
+#define BMX055_ACC_ADDR_DEFAULT  0x18
+#define BMX055_GYRO_ADDR_DEFAULT 0x68
```

Comment:
minor style: its good practice to put parenthesis around scalar values in defines